pages/pipelines/service_quotas.md

```diff
@@ -0,0 +1,97 @@
+---
+toc: false
+---
+
+# Service quotas
+
+> 📘 Need a limit increased?
+> You can request a limit increase by [contacting support](mailto:support@buildkite.com), and providing details about your use case.
+
+Service quotas are put in place to ensure that Buildkite can provide a reliable service to all customers. These quotas are scoped to your organization, and can be increased by contacting support and providing details about your use case.
+
+<table>
+  <tbody>
+    <% [
+      {
+        title: "Invitations per organization",
+        description: "The maximum number of pending invitations for an organization",
+        default_value: "20 invitations"
```

Comment:
Another general thought – how do we keep these values synced with what's in bk/bk?

Reply:
An assertion spec in bk/bk, with a link to this guide doc to update it if an engineer changes the default?

Reply:
We might be able to leverage some internal tooling options (e.g. https://github.com/danger/danger) to help us out here in the bk/bk side 👀